services/api/src/submission/checks/base-checks/color-contrast-greater-than-or-equal.check.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable, Logger } from "@nestjs/common";
+import ColorContrastChecker from "color-contrast-checker";
+import { ColorTranslator } from "colortranslator";
+import { DOMWindow } from "jsdom";
+
+import { CodeLevelSubmission as Submission } from "../../graphql/models/code-level-submission.model";
+import { Rule } from "../../interfaces/rule.interface";
+import { RuleCheckResult } from "../../interfaces/rule-check-result.interface";
+import { JsdomCheck } from "./jsdom-check";
+
+@Injectable()
+export class ColorContrastGreaterThanOrEqual extends JsdomCheck {
+  constructor(logger: Logger) {
+    super(logger);
+  }
+
+  public async evaluateRule(window: DOMWindow, submission: Submission, rule: Rule): Promise<RuleCheckResult> {
+    for (const option of ["selector", "minContrastValue", "property1", "property2"]) {
+      if (!this.ruleHasOption(rule, option)) {
+        return this.checkConfigurationError(submission, rule, option);
+      }
+    }
+
+    const { selector, minContrastValue, property1: cssProperty1, property2: cssProperty2 } = rule.options;
```

Comment:
Why do you rename the properties while restructuring the options here? In my opinion keeping the names the same between the options object and the code makes it easier to reason about the code.

Reply:
I copied the code from the other css checks and just adjusted it. That's the way it's done in both other css checks so I wanted to stay consistent:

https://github.com/a11yphant/a11yphant/blob/8ff010f6df06d80ce9f2ede90ced1db00a692667/services/api/src/submission/checks/base-checks/element-has-css-property-value.check.ts#L22

https://github.com/a11yphant/a11yphant/blob/8ff010f6df06d80ce9f2ede90ced1db00a692667/services/api/src/submission/checks/base-checks/element-has-css-property-value-greater-than.check.ts#L22